lib/Hakyll/Core/Compiler/Internal.hs

```diff
@@ -132,7 +135,7 @@ data CompilerResult a
     = CompilerDone a CompilerWrite
     | CompilerSnapshot Snapshot (Compiler a)
     | CompilerRequire (Identifier, Snapshot) (Compiler a)
-    | CompilerError (Reason [String])
+    | CompilerError (CompilerErrors String)
```

Comment:
This is why I did use the name `Reason`, `(Compiler)Errors` looks just weird here.